docs/tutorials/studio-tutorial.md

```diff
@@ -572,6 +572,15 @@ This panel lets you define tests for the services you write by specifying test d
 
 ## Create a Service Test 
 
+### Create connections test data
+Click the `generate` button to automatically generate test data if possible. Otherwise, please manually create test data.
```

Comment:
Maybe we should also mention something around when should a particular Data Type be used. 
Current strategy for defining subtypes is 
1 for each store types
1 for externalFormats
1 for reference